extras/rya.console/src/main/java/mvm/rya/shell/RyaConnectionCommands.java

```diff
@@ -35,13 +35,17 @@
 import org.springframework.stereotype.Component;
 
 import com.google.common.base.Optional;
+import com.google.common.collect.Lists;
 import com.mongodb.MongoClient;
+import com.mongodb.MongoCredential;
+import com.mongodb.ServerAddress;
 
 import mvm.rya.shell.SharedShellState.ConnectionState;
 import mvm.rya.shell.command.CommandException;
 import mvm.rya.shell.command.RyaCommands;
 import mvm.rya.shell.command.accumulo.AccumuloConnectionDetails;
 import mvm.rya.shell.command.administrative.InstanceExists;
+import mvm.rya.shell.command.mongo.MongoConnectionDetails;
```

Comment:
Also, RyaAdminCommands is going to need to be updated to only show the supported commands. Since Mongo DB will not support add/remove PCJ, then they shouldn't be presented as options to the user.